src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -0,0 +1,436 @@
+        :documentOptionalEmail="getDocumentDelivery.documentOptionalEmail"
+        @update:optionalEmail="setDocumentOptionalEmail($event)"
+        @valid="setDocumentOptionalEmailValidity($event)"
+      />
+      </v-card>
+    </section>
+
+    <!-- Folio or Reference Number -->
+    <section id="folio-number-section" class="mt-10" v-if="isPremiumAccount">
+      <header>
+        <h2>Folio or Reference Number for this Filing</h2>
+        <p class="mt-4 mb-6">
+          Enter the folio or reference number you want to use for this filing for your own tracking
+          purposes. The Business Folio or Reference Number is displayed below (if available).
+          Entering a different value below will not change the Business Folio or Reference Number.
+          Only the number below will appear on the transaction report and receipt for this filing.
+        </p>
+      </header>
+       <v-card flat class="mt-6">
+        <TransactionalFolioNumber
+          :accountFolioNumber="getFolioNumber"
+          :transactionalFolioNumber="getTransactionalFolioNumber"
+          :doValidate="getValidateSteps"
+          @change="setTransactionalFolioNumber($event)"
+          @valid="setTransactionalFolioNumberValidity($event)"
+        />
+       </v-card>
+    </section>
+  <!-- Completing Party -->
+  <!-- show only if exist -->
+    <section  id="completing-party-section" class="mt-10"
+    v-if="getAddPeopleAndRoleStep.orgPeople && getAddPeopleAndRoleStep.orgPeople.length > 0">
+      <header>
+        <h2>Completing Party</h2>
+
+      </header>
+      <v-card flat class="mt-6">
+        <ListPeopleAndRoles
+        :personList="getAddPeopleAndRoleStep.orgPeople"
+        :isSummary="true"
+        :showErrorSummary="!getAddPeopleAndRoleStep.valid"
+      />
+      </v-card>
+    </section>
+
+    <!-- Certify -->
+    <section id="certify-section" class="mt-10">
+      <header>
+        <h2>Certify</h2>
+        <p class="mt-4 mb-6">
+          Confirm the legal name of the person authorized to complete and submit this dissolution.
+        </p>
+      </header>
+      <v-card flat class="mt-6">
+      <Certify
+        :class="{ 'invalid-section': isCertifyInvalid }"
+        :currentDate="getCurrentDate"
+        :certifiedBy="getCertifyState.certifiedBy"
+        :entityDisplay="getCompletingPartyStatement.entityDisplay"
+        :isCertified="getCertifyState.valid"
+        :statements="getCompletingPartyStatement.certifyStatements"
+        :message="getCompletingPartyStatement.certifyClause"
+        :isStaff="isRoleStaff"
+        :firstColumn="3"
+        :secondColumn="9"
+        :invalidSection="isCertifyInvalid"
+        :disableEdit="!isRoleStaff"
+        @update:certifiedBy="onCertifiedBy($event)"
+        @update:isCertified="onIsCertified($event)"
+      />
+      </v-card>
+    </section>
+
+    <!-- Court Order and Plan of Arrangement -->
+    <section id="court-order-poa-section" class="mt-10" v-if="isRoleStaff">
```

Comment:
Please ask BA/PO if this section applies to SBC Staff also.

Re: https://github.com/bcgov/business-create-ui/blob/main/src/store/getters/state-getters.ts#L93

Reply:
This might also apply to the Staff Payment section.

Reply:
OK.

(I saw todays that there is a Design ticket to look at changes for staff filings in Create UI... so it's coming.)